src/layout/dom.ts

```diff
@@ -171,6 +173,26 @@ export function update(node: Node, parent: Node, data: NodeInfo, source: Source)
     }
 }
 
+export function sameorigin(node: Node): boolean {
+    let output = false;
+    if (node.nodeType === Node.ELEMENT_NODE && (node as HTMLElement).tagName === Constant.IFRAME_TAG) {
+        let frame = node as HTMLIFrameElement;
```

Comment:
>HTMLIFrameElement [](start = 28, length = 17)

When could this error happen? Add a brief comment. I assume it is in cases where not same origin causing a security exception?

Reply:
Yes thanks, will add comment. When it's cross-origin, browser will throw exception.